### PR TITLE
#408 Use lowercase in cmake instructions.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/3rdParty/libzip/FindlibZIP.cmake [hunk 1]
```diff
@@ -15,53 +15,79 @@
...
+  # (Use ${CMAKE_ROOT}/Modules instead of ${CMAKE_CURRENT_LIST_DIR} because CMake
+  #  itself includes this FindlibZIP when built with an older CMake that does
+  #  not provide it.  The older CMake also does not have CMAKE_CURRENT_LIST_DIR.)
+  include(FindPackageHandleStandardArgs)
+  find_package_handle_standard_args(libZIP
+    REQUIRED_VARS libZIP_LIBRARY libZIP_INCLUDE_DIR
+    VERSION_VAR libZIP_VERSION)
+endif()
+
+if(libZIP_FOUND)
```
Comment (rosiereflo) [inline dynawo/3rdParty/libzip/FindlibZIP.cmake:71]: This code wasn't here before if I'm not mistaken. When a coding style change creates so many differences, it is always better to not add new code (even if seems harmless). Please put this change in another PR (along with the one in libArchive)

File: dynawo/3rdParty/libzip/cmake/FindLibArchive.cmake [hunk 2]
```diff
@@ -15,45 +15,64 @@
...
-UNSET(LIBARCHIVE_FOUND)
-
-IF (LibArchive_FOUND)
-    SET(LibArchive_INCLUDE_DIRS ${LibArchive_INCLUDE_DIR})
-    SET(LibArchive_LIBRARIES ${LibArchive_LIBRARY})
-ENDIF ()
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(LibArchive DEFAULT_MSG LibArchive_LIBRARY LibArchive_INCLUDE_DIR)
+set(LibArchive_FOUND ${LIBARCHIVE_FOUND})
+UNset(LIBARCHIVE_FOUND)
```
Comment (rosiereflo) [inline dynawo/3rdParty/libzip/cmake/FindLibArchive.cmake:54]: unset

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/3rdParty/libzip/FindlibZIP.cmake (hunk 1 above)
Comment (gautierbureau) [reply dynawo/3rdParty/libzip/FindlibZIP.cmake:71]: in fact those changes are just copy/paste of the same file in dynawo/cmake, we had the same files but not containing the same code so I just corrected this. so it is not really new code

File: dynawo/3rdParty/libzip/cmake/FindLibArchive.cmake (hunk 2 above)
Comment (gautierbureau) [reply dynawo/3rdParty/libzip/cmake/FindLibArchive.cmake:54]: done

**Review by rosiereflo — APPROVED** [review]
